app/Http/Controllers/API/User/AuthController.php

```diff
@@ -28,7 +28,16 @@ public function register(UserRegisterRequest $request)
         $data = $request->all();
         $data['password'] = Hash::make($request->password);
         if (User::create($data)) {
-            return $this->successResponse(User::latest()->first(), Response::HTTP_CREATED);
+            $user = User::latest()->first();
```

Comment:
Em có thể xem code của method `latest()` ở đây: https://github.com/laravel/framework/blob/5.6/src/Illuminate/Database/Query/Builder.php#L1693
Theo anh nó không hoàn toàn đúng khi em muốn lấy user vừa create.
Thay vì dùng `create()` em có thể kết hợp `make()` và `save()` để dùng. khi `save()` thì sẽ có object user vừa create luôn.